_terms/legal-privacy-policy.md

```diff
@@ -117,10 +122,15 @@ Lawful processing of your personal data is ensured by applying the following:
 
 Where do we keep your personal data?
 
-All personal data is kept within EU and Swiss data centres and might be moved between data centres residing in other countries also within the EU and Switzerland. Personal data is protected according to the [Code of Conduct for Service Providers](https://geant3plus.archive.geant.net/uri/dataprotection-code-of-conduct/v1/Pages/default.aspx), a common standard for the research and higher education sector to protect your privacy.
+All personal data is kept within EU and Swiss data centres and might be moved between data centres residing in other countries also within the EU and Switzerland.
 
 
 4\. Applicability 
 ---
 
 Independently of the existing relations with third-party partners and references to other websites, this privacy notice only applies to SixSq's websites and web services like [sixsq.com](https://sixsq.com), [Nuvla.io](https://nuvla.io) and any other public pages with static content, owned by SixSq and related to our services and products. This includes the documentation website [docs.nuvla.io](https://docs.nuvla.io).
+
+5\. Right to forget
+---
+
+If you want us to delete all the private information we have collected from you, please send us an email at: [support@sixsq.com](mailto:support@sixsq.com?subject=Delete my personal information).  All you private information will be deleted from our services within 7 days.
```

Comment:
what about the 3rd party services?

Reply:
You mean Intercom and Stripe?

Reply:
yes...can they ask us to delete it? They should...

Reply:
Done

Reply:
Done